jerry-core/ecma/operations/ecma-array-object.c

```diff
@@ -333,6 +344,48 @@ ecma_fast_array_extend (ecma_object_t *object_p, /**< fast access mode array obj
   return new_values_p;
 } /* ecma_fast_array_extend */
 
+/**
+ * Find the given indexed property, if the search fails, it will convert back the fast-array
```

Comment:
Perform [[Get]] operation for fast access mode arrays with the given indexed property.
Note:
 - The given index must be less than the array's length
 - Since the property lookup affect the object's in the prototype chain the fast array may converted back to regular array object. The caller must check whether the array is still fast mode array before reaccessing the internal value buffer.
 - Returned value increases the reference count of non-object types.
 - If the property is not found ECMA_VALUE_ARRAY_HOLE is returned